apps/asap-server/src/utils/assets.ts

```diff
@@ -0,0 +1,10 @@
+import { cms } from '../config';
+
+export const createURL = (assets: string[]): string | undefined => {
+  if (assets.length === 0) {
+    return undefined;
+  }
+
+  const asset = assets[0];
+  return `${cms.baseUrl}/api/assets/${cms.appName}/${asset}`;
```

Comment:
idk what `cms.baseUrl` contains, if it's something like `https://my.squidex.com/` then this might be better to deal slash / no slash
```suggestion
  return new URL(`/api/assets/${cms.appName}/${asset}`, cms.baseUrl);
```